tests/figures/chart/bar_chart_plugin.test.ts

```diff
@@ -132,4 +132,64 @@ describe("bar chart", () => {
       expect(runtime.chartJsConfig.data.datasets[0].yAxisID).toBe("y");
     });
   });
+
+  test("Bar chart legend", () => {
+    const model = new Model({
+      sheets: [
+        {
+          name: "Sheet1",
+          colNumber: 10,
+          rowNumber: 10,
+          rows: {},
+          cells: {
+            A1: { content: "1" },
+            A2: { content: "2" },
+            A3: { content: "3" },
+            A4: { content: "4" },
+          },
+        },
+      ],
+    });
```

Comment:
```suggestion
    const model = createModelFromGrid({A1: "1", ... });
    });
```
way more readable IMO

Reply:
Actually do you care at all about the data in A1:A4 ?

Reply:
I don't care about the value, but I need some values in A1:A4 to get a valid chart/runtime